plugins/olm-team/commands/ep-watch.md

```
@@ -0,0 +1,353 @@
+---
+description: Watch open Enhancement PRs from other teams that may impact OLM
+argument-hint: ""
+---
+
+## Name
+olm-team:ep-watch
+
+## Synopsis
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)